schema_based_on_diagram.sql

```diff
@@ -0,0 +1,22 @@
+-- Active: 1693915950566@@127.0.0.1@7499
+CREATE DATABASE clinic;
+CREATE TABLE patients(id SERIAL PRIMARY KEY, name VARCHAR(40), date_of_birth DATE);
+CREATE TABLE medical_histories(ID INT, admitted_at TIMESTAMP, patient_id INT, status VARCHAR(90),
+PRIMARY KEY(ID),
+FOREIGN KEY(patient_id) REFERENCES patients(id));
+SELECT * FROM medical_histories;
+CREATE TABLE treatments(id INT PRIMARY KEY,  type VARCHAR(60), name VARCHAR(50));
+
+-- Now a talbe to join the id of treatment and medical history
+CREATE TABLE medical_treatment (
+    medical_history_id INT,
+    treatment_id INT,
+    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(ID),
+    FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+
```

Comment:
- [ ] It would be better if we create indexes for the foreign keys; for example;`patient_id`, `invoice_id`, `treatment_id`, `medical_history_id`. Here is how we can implement it `CREATE INDEX ON medical_histories (patient_id);` kindly do that for the other tables
![Screenshot from 2023-09-05 22-55-14](https://github.com/najibullahjafari/vet-clininc/assets/46388113/4a4feee0-75d3-4db1-883b-462ca3d11700)